### PR TITLE
Added people-rw-neo4j serivce.

### PR DESCRIPTION
**Pull request by carlosroman**

### CODE REVIEW

File: people-rw-neo4j-sidekick@.service [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+BindsTo=people-rw-neo4j@%i.service
+After=people-rw-neo4j@%i.service
+
+[Service]
+ExecStart=/bin/sh -c "\
+  etcdctl set   /ft/services/people-rw-neo4j/healthcheck     true; \
+  etcdctl mkdir /ft/services/people-rw-neo4j/servers; \
+  etcdctl set /ft/healthcheck/people-rw-neo4j-%i/path /__health; \
+  while true; do \
+    CONTENT_RW_PORT=`[ $CONTENT_RW_PORT ] && echo $CONTENT_RW_PORT || echo $(/usr/bin/docker port people-rw-neo4j-%i 8080 | cut -d':' -f2)`; \
```
Comment (tosan88) [inline people-rw-neo4j-sidekick@.service]: Can you please change to be PEOPLE_RW_PORT or simply PORT?

File: people-rw-neo4j@.service [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+KillMode=none
+ExecStartPre=-/bin/bash -c '/usr/bin/docker kill %p-%i > /dev/null 2>&1'
+ExecStartPre=-/bin/bash -c '/usr/bin/docker rm %p-%i > /dev/null 2>&1'
+ExecStartPre=/bin/bash -c '/usr/bin/docker history coco/people-rw-neo4j:$DOCKER_APP_VERSION > /dev/null 2>&1 || /usr/bin/docker pull coco/people-rw-neo4j:$DOCKER_APP_VERSION'
+
+ExecStart=/bin/sh -c '\
+  export APP_PORT=8080; \
+  export ENV=$(/usr/bin/etcdctl get /ft/config/environment_tag); \
+  export NEO_HOST_PORT=$(/usr/bin/etcdctl get /ft/services/neo4j/servers/1); \
+  /usr/bin/docker run --rm --name %p-%i -p $APP_PORT --env "NEO_URL=$NEO_HOST_PORT/db/data" --env "APP_PORT=$APP_PORT" --env "BATCH_SIZE=50" --env "GRAPHITE_ADDRESS=graphite.ft.com:2003" --env "GRAPHITE_PREFIX=coco.services.$ENV.people-rw-neo4j.%i" coco/people-rw-neo4j:$DOCKER_APP_VERSION;'
```
Comment (tosan88) [inline people-rw-neo4j@.service]: Hmm, why the small batch-size?

File: people-rw-neo4j-sidekick@.service (hunk 1 above)
Comment (martingartonft) [inline people-rw-neo4j-sidekick@.service]: PORT is typical. It makes copy-paste mistakes (like this one) less likely if we stay generic.

File: people-rw-neo4j@.service (hunk 2 above)
Comment (carlosroman) [inline people-rw-neo4j@.service]: Copy and paste without changing stuff.

**Comment by tosan88** [comment]
:+1:

**Pull request merged**